src/GafferImage/ImageWriter.cpp

```diff
@@ -705,7 +705,7 @@ ImageWriter::ImageWriter( const std::string &name )
 	storeIndexOfNextChild( g_firstPlugIndex );
 	addChild( new ImagePlug( "in" ) );
 	addChild( new StringPlug( "fileName" ) );
-	addChild( new StringPlug( "channels", Gaffer::Plug::In, "[RGB]" ) );
+	addChild( new StringPlug( "channels", Gaffer::Plug::In, "*" ) );
```

Comment:
Cool. I think the original default was conservative because OIIO would fail to write things like JPEGs if we threw too many channels at it, but for a while now we've been testing `out->supports( "channels" )` and `out->supports( "alpha" )` to avoid those problems anyway.